Apps/W1/HybridGP/app/src/codeunits/GPCloudMigration.codeunit.al

```diff
@@ -139,14 +132,42 @@ codeunit 4025 "GP Cloud Migration"
     local procedure CreateDataMigrationEntites(var DataMigrationEntity: Record "Data Migration Entity"): Boolean
     var
         HelperFunctions: Codeunit "Helper Functions";
+        GPCompanyAdditionalSettings: Record "GP Company Additional Settings";
     begin
         DataMigrationEntity.InsertRecord(Database::"G/L Account", HelperFunctions.GetNumberOfAccounts());
-        DataMigrationEntity.InsertRecord(Database::Customer, HelperFunctions.GetNumberOfCustomers());
-        DataMigrationEntity.InsertRecord(Database::Vendor, HelperFunctions.GetNumberOfVendors());
-        DataMigrationEntity.InsertRecord(Database::Item, HelperFunctions.GetNumberOfItems());
+
```

Comment:
Are these record counts valid based upon selections made in the UI?

Reply:
Yes, Customers and Vendors use the SQL query to filter out inactives and the counts are based on what ends up in the staging tables. There is no item filtering yet, but it will be in the next feature and the count issue has been accounted for there.